src/diffusers/pipelines/wuerstchen/__init__.py

```diff
@@ -0,0 +1,9 @@
+from ...utils import is_torch_available, is_transformers_available
+
+
+if is_transformers_available() and is_torch_available():
+    from .diffnext import DiffNeXt
```

Comment:
Where does the name `DiffNeXt` come from? Should we maybe call it `WuerstchenDiffNeXt`?